### ISSUE TITLE
Some browser name is missing from the collected data

### ISSUE THREAD

**Issue by tpimh** [plausible/analytics#2524]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

After an upgrade to 1.5.1 according to the migration guide (analytics, clickhouse and postgres upgraded), I noticed that one of the browsers lost its name. Now it's shown as an empty bar as seen on the screenshot below.

This issue is not critical. It's rather an unpopular browser (if it was chrome, firefox or safari, I would've cared much), and I don't know what it showed as before upgrade, but I definitely recall that there were no empty bars under the browsers. I would be okay with just removing it from the statistics, but I don't know how to do it.

### Expected behavior

The browser name (I don't know what it was), should be kept between upgraded.

### Screenshots

<img width="519" alt="image" src="https://user-images.githubusercontent.com/1071872/207910789-fc6f71c0-0dd8-4436-9d93-99b788795fb6.png">

### Environment

```markdown
- OS: Alpine Linux
- Browser: Firefox (reproducible on other browsers)
- Browser Version: 107 (reproducible on other versions)
```

**Comment by ruslandoga** [comment]
@tpimh 👋

I wonder if that browsers shows up in the CSV export?

**Comment by tpimh** [comment]
Thanks for your quick reaction! Nope, in CSV export it's line shows as this: `,2`.

**Comment by ruslandoga** [comment]
There seems to be an empty browser name on https://plausible.io/plausible.io/browsers?period=all as well, at the very bottom.

**Comment by tpimh** [comment]
@ruslandoga I just noticed that there's also a size with no name, that's interesting.

**Comment by ukutaht** [comment]
The empty name means that the browser could not be determined. We now show an explicit `(not set)` entry instead of an empty line to make it clearer. Fixed in #2646
